ginga/examples/configs/plugin_Zoom.cfg

```diff
@@ -3,21 +3,10 @@
 #
 # Place this in file under ~/.ginga with the name "plugin_Zoom.cfg"
 
-# default zoom radius in pixels
-zoom_radius = 30
-
 # default zoom level
 zoom_amount = 3
 
 # refresh interval (sec)
 # NOTE: usually a small delay speeds things up
 refresh_interval = 0.02
 
-# rotate the zoom image if the channel image is rotated?
```

Comment:
Hmm, so all the removed settings are basically features that are no longer supported?

Reply:
Yes.  There is actually no reason that the new version of the plugin could not support those features.  Since they were turned off by default I don't know how many people were using them anyway.  If we get an issue that someone misses one of those features it would be easy enough to add it back in.  But the plugin is simpler without them.